app/react/Review/components/OneUpEntityViewer.tsx

```diff
@@ -185,7 +184,7 @@ export class OneUpEntityViewerBase extends Component<
                           showType={false}
                           showSubset={this.nonMlProps()}
                         />
-                        <FileList files={entity.documentsdocuments} entity={entity} />
+                        <FileList files={entity.documents} entity={entity} />
```

Comment:
Here, I had to just assume that `documentsdocuments` is a typo.  No idea if that ever worked.  We have NOTHING in the entire system that assigns a `documentsdocuemnts` property. Still, I cannot test this properly, so it is just an assumption.

Reply:
this is 100% a typo